src/qt/ve_qitems_mqtt.cpp

```diff
@@ -950,7 +970,8 @@ bool WebSocketDevice::open(QIODeviceBase::OpenMode mode)
 	QWebSocketHandshakeOptions options;
 	options.setSubprotocols(QStringList { QString::fromUtf8(mProtocol) });
 	mWebSocket.open(mUrl, options);
-	return QIODevice::open(mode);
+	mConnecting = QIODevice::open(mode);
```

Comment:
it would be good to log something if QIODevice::open() fails and returns false

Reply:
good point, will do

Reply:
done, handled by the caller